datasets/jigsaw_unintended_bias/README.md

```diff
@@ -0,0 +1,190 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc0-1.0
+multilinguality:
+- monolingual
+pretty_name: Jigsaw Unintended Bias in Toxicity Classification
+size_categories:
+- 1M<n<10M
+source_datasets:
+- original
+task_categories:
+- text-scoring
+task_ids:
+- text-scoring-other-toxicity-prediction
+---
+
+# Dataset Card for Jigsaw Unintended Bias in Toxicity Classification
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage: https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification **
+- **Repository: N/A **
+- **Paper: N/A **
+- **Leaderboard: https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification/leaderboard **
+- **Point of Contact: N/A **
+
+### Dataset Summary
+
+The Jigsaw Unintended Bias in Toxicity Classification dataset comes from the eponymous Kaggle competition.
+
+Please see the original [data](https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification/data)
+ description for more information.
+
+### Supported Tasks and Leaderboards
+
+The main target for this dataset is toxicity prediction. Several toxicity subtypes are also available, so the dataset 
+can be used for multi-attribute prediction.
+
+See the original [leaderboard](https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification/leaderboard)
+for reference.
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+A data point consists of an id, a comment, the main target, the other toxicity subtypes as well as identity attributes.
```

Comment:
Could you add an actual sample from the dataset ?

You can get one by using `load_dataset` and getting the first one `dataset["train"][0]` for example

Reply:
Done